packages/base/cards-grid.gts

```diff
@@ -269,6 +273,71 @@ class Isolated extends Component<typeof CardsGrid> {
       realmURL: this.args.model[realmURL],
     });
   });
+
+  private loadFilterList = restartableTask(async () => {
+    let response = await fetch(`${this.realms[0]}_types`, {
+      headers: {
+        Accept: SupportedMimeType.CardTypeSummary,
+      },
+    });
+    if (!response.ok) {
+      let responseText = await response.text();
+      let err = new Error(
+        `status: ${response.status} -
+          ${response.statusText}. ${responseText}`,
+      ) as any;
+
+      err.status = response.status;
+      err.responseText = responseText;
+
+      throw err;
+    }
+    let cardTypeSummaries = (await response.json()).data as {
+      id: string;
+      attributes: { displayName: string; total: number };
+    }[];
+    cardTypeSummaries.forEach((summary) => {
+      if (summary.attributes.displayName === 'Cards Grid') {
+        return;
+      }
+      const lastIndex = summary.id.lastIndexOf('/');
+      this.filters.push({
+        displayName: summary.attributes.displayName,
+        query: {
+          filter: {
+            type: {
+              module: summary.id.substring(0, lastIndex),
+              name: summary.id.substring(lastIndex + 1),
+            },
+          },
+          sort: [
+            {
+              on: {
+                module: `${baseRealm.url}card-api`,
+                name: 'CardDef',
+              },
+              by: '_cardType',
+            },
+            {
+              on: {
+                module: `${baseRealm.url}card-api`,
+                name: 'CardDef',
+              },
+              by: 'title',
+            },
+          ],
+        },
+      });
+    });
+  });
+
+  private get hideCards() {
+    return (
+      this.activeFilter.displayName === 'Favorites' &&
+      (!this.args.model['favorites'] ||
```

Comment:
Is there a reason this can't be `this.args.model.favorites` rather than the string index style?